lib/puppet/functions/classified_with.rb

```diff
@@ -0,0 +1,9 @@
+Puppet::Functions.create_function(:classified_with) do
```

Comment:
Should these functions be named `is_classified_with`?

Are these parse-order safe?